lmdeploy/pytorch/engine/model_agent.py

```diff
@@ -1247,6 +1254,25 @@ async def async_forward(self, inputs: ModelInputs, swap_in_map: SwapMap,
         return output
 
 
+# [Note] Exit By Sending Exit Flag
```

Comment:
Let's put the comments in the function as the docstring. 
Here is an example of docstring: https://github.com/InternLM/lmdeploy/blob/32c858055c58fe528cdc1be157dd646af3be6f53/lmdeploy/api.py#L96